src/models.js

```diff
@@ -304,6 +272,34 @@ module.exports = cooler => {
     });
   };
 
+  const socialFilter = async ({
+    following = null,
+    blocking = false,
+    me = false
+  } = {}) => {
```

Comment:
A comment could be nice here to describe that `null` means don't apply a filter, `true`/`false` mean apply a filter for that value respectively?

Also if omitted when calling the function they get these as default values?  So the function doesn't need to deal with them being undefined?

(I think this makes sense, I'm just not familiar with unpacking an object and setting defaults in the function signature like this.  I don't know what the `= {}` is doing on 279)